athenz-identity-provider-service/src/main/java/com/yahoo/vespa/hosted/athenz/instanceproviderservice/ca/model/SignedCertificate.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.athenz.instanceproviderservice.ca.model;
```

Comment:
Why "model"?
This is a serialization object - nobody should ever use it in their internal logic (because mixing these things means that refactoring of the business logic ("model") will break the REST API) - so it should be in a package named "restapi" or similar, and preferably be named something like "CertificateSerializedPayload" or similar, such that people won't misuse it, which so far has happened with every single class like this we have ever made  :-)

Reply:
Renamed the class, but I'll keep the `model` for now for consistency within the sub-project, changing that would require a lot of moving and should be done in separate PR. This is also just a temporary REST API until we can get the JDisc filters to work.

Reply:
I don't think it is worth the extra boilerplate when business logic is relatively small. Also, we are using Jackson properly (e.g using `@JsonProperty` for defining the name of serialized fields), which was the typical issue for earlier Jackson "incidents".